metabase/metabase
# sync snowplow dashboard schemas

### Description

We've added a few new events to the `dashboard` schema and bumped it to `1-1-1` but it has not been published. Then we added one more (pdf downloads) and created `1-1-2`. So since `1-1-1` has never been published, the changes from `1-1-2` got published under `1-1-1`. This PR updates schemas in our repo accordingly

[Convo](https://metaboat.slack.com/archives/C04FZTRF457/p1688422264374709)

### How to verify

- check schema has all needed `dashboard` events
- check the tracking code passes the right schema version
